src/main/resources/db/migration_h2/V4__foreign_keys.sql

```diff
@@ -0,0 +1,245 @@
+
+alter table "asset"
+	add constraint ASSET_ASSET_ID_FK
+		foreign key ("account_id") references "asset" ("id")
+			on delete cascade;
+
+-- asset_transfer
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ACCOUNT_ID_FK
+		foreign key ("sender_id") references "account" ("id")
+			on delete cascade;
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ACCOUNT_ID_FK_2
+		foreign key ("recipient_id") references "account" ("id")
+			on delete cascade;
+
+alter table "asset_transfer"
+	add constraint ASSET_TRANSFER_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+-- at
+
+alter table "at"
+	add constraint AT_ACCOUNT_ID_FK
+		foreign key ("creator_id") references "account" ("id")
+			on delete cascade;
+
+-- at_state
+
+alter table "at_state"
+	add constraint AT_STATE_AT_ID_FK
+		foreign key ("at_id") references "at" ("id")
+			on delete cascade;
+
+-- bid_order
+
+alter table "bid_order"
+	add constraint BID_ORDER_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "bid_order"
+	add constraint BID_ORDER_ASSET_ID_FK
+		foreign key ("asset_id") references "asset" ("id")
+			on delete cascade;
+
+-- block
+-- no generator_id foreign key because account may be not yet created
+
+-- escrow
+
+alter table "escrow"
+	add constraint ESCROW_ACCOUNT_ID_FK
+		foreign key ("sender_id") references "account" ("id")
+			on delete cascade;
+
+alter table "escrow"
+	add constraint ESCROW_ACCOUNT_ID_FK_2
+		foreign key ("recipient_id") references "account" ("id")
+			on delete cascade;
+
+-- escrow_decision
+
+alter table "escrow_decision"
+	add constraint ESCROW_DECISION_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "escrow_decision"
+	add constraint ESCROW_DECISION_ESCROW_ID_FK
+		foreign key ("escrow_id") references "escrow" ("id")
+			on delete cascade;
+
+-- goods
+
+alter table "goods"
+	add constraint GOODS_ACCOUNT_ID_FK
+		foreign key ("seller_id") references "account" ("id")
+			on delete cascade;
+
+-- indirect_incoming
+
+alter table "indirect_incoming"
+	add constraint INDIRECT_INCOMING_ACCOUNT_ID_FK
+		foreign key ("account_id") references "account" ("id")
+			on delete cascade;
+
+alter table "indirect_incoming"
+	add constraint INDIRECT_INCOMING_TRANSACTION_ID_FK
+		foreign key ("transaction_id") references "transaction" ("id")
+			on delete cascade;
+
+-- purchase
+
+alter table "purchase"
+	add constraint PURCHASE_ACCOUNT_ID_FK
+		foreign key ("buyer_id") references "account" ("id")
+			on delete cascade;
+
+alter table "purchase"
+	add constraint PURCHASE_ACCOUNT_ID_FK_2
+		foreign key ("seller_id") references "account" ("id")
+			on delete cascade;
+
+alter table "purchase"
+	add constraint PURCHASE_GOODS_ID_FK
+		foreign key ("goods_id") references "goods" ("id")
+			on delete cascade;
+
+-- purchase_feedback
+-- TODO: what is purchase_feedback.id? purchase.id?
```

Comment:
question

Reply:
Purchase Feedback's DB id = purchase's ID = ID of transaction where purchase was made, yes.

Reply:
https://github.com/burst-apps-team/burstcoin/blob/c9af1999647f04f9e876e3cb6b3f03653eacd733/src/main/kotlin/brs/db/sql/SqlDigitalGoodsStoreStore.kt#L18

Reply:
thx will add